src/nl/hannahsten/texifyidea/formatting/TexSpacingBuilder.kt

```diff
@@ -13,7 +13,7 @@ import com.intellij.psi.tree.TokenSet
  *
  * @author Sten Wessel
  */
-class LatexSpacingBuilder(private val commonSettings: CommonCodeStyleSettings) {
+class TexSpacingBuilder(private val commonSettings: CommonCodeStyleSettings) {
```

Comment:
Why the change?

Reply:
To avoid confusion; it is used by both `LatexFormattingModelBuilder` and `BibtexFormattingModelBuilder`.